packages/annotations/src/block/index.js

```diff
@@ -28,8 +32,4 @@ const addAnnotationClassName = ( OriginalComponent ) => {
 	} )( OriginalComponent );
 };
 
-addFilter(
-	'editor.BlockListBlock',
-	'core/annotations',
-	addAnnotationClassName
-);
+addFilter( 'editor.BlockListBlock', STORE_NAME, addAnnotationClassName );
```

Comment:
This one is probably incidentally the same. It should remain hardcoded since it need to always remain the same fir backward compatibility.

Reply:
Sorry for taking so long to answer. Shouldn't it be ok, as the change was using the store name (constant) instead of the definition?

Reply:
I fixed it, conceptually those are 2 different things.

Reply:
You mean that this `core/annotations` reference shouldn't be replaced since it's not related to the store definition? 
If that's the case, then I'm sorry. I didn't go further on the function to check whether it was related or not, just replaced it. Sorry :(